### PR TITLE
Umbraco Cloud release notes for March

### PR DESCRIPTION
**Pull request by sajumb**
## Description

Added the page for Umbraco Cloud release notes for March

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ X ] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)
As soon as possible

_When should the content be published?_

### CODE REVIEW

**Comment by alina-tincas** [comment]
Hi @sajumb thank you for the PR 🙌 

We will review it as soon as we can 🙏

File: umbraco-cloud/release-notes/2024-03-releasenotes.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# March 2024
+
+## Key Takeaways
+
+* **Database Export** - Export your database smoothly with point-in-time recovery, ensuring your project's data is always safeguarded for when you need it most.
+
+## Export Your Database Smoothly With Point-In-Time Recovery.
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2024-03-releasenotes.md:7]:
```suggestion
## Export Your Database Smoothly With Point-In-Time Recovery
```

**Comment by sajumb** [comment]
Hey! Any update on when this will be merged to main?

File: umbraco-cloud/release-notes/2024-03-releasenotes.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# March 2024
+
+## Key Takeaways
+
+* **Database Export** - Export your database smoothly with point-in-time recovery, ensuring your project's data is always safeguarded for when you need it most.
+
+## Export Your Database Smoothly With Point-In-Time Recovery
+With the introduction of the Point-In-Time Recovery (PITR) feature on the new "Backups" page, managing backups of your cloud projects database has never been more straightforward. This new feature empowers you with the flexibility to export backups of your Umbraco database directly from Azure on your terms, perfect for those moments right before a significant update, during troubleshooting, or when you need to revert recent changes.
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2024-03-releasenotes.md:8]:
```suggestion
With the introduction of the Point-In-Time Recovery (PITR) feature on the new "Backups" page, managing backups of your cloud projects database has never been more straightforward. This new feature empowers you with the flexibility to export backups of your Umbraco database directly from Azure on your terms. Perfect for those moments right before a significant update, during troubleshooting, or when you need to revert recent changes.
```

**Review by jonat123 — APPROVED** [review]
@sajumb Will merge it in now :-)

**Pull request merged**